src/App.tsx

```diff
@@ -1,5 +1,29 @@
+import colors from './colors.json';
+
+const ColorsGreeting = ({ name, hex, comp }) => {
+    return (
+        <div>
+            <h1 style={{ color: `#${hex}`, padding: '2em 2em 0em 2em', fontSize: '2em', display: 'flex', justifyContent: 'center' }}>Hello human, I am {name}.</h1>
+            <p style={{ color: 'white', fontSize: '1.2em', display: 'flex', justifyContent: 'center' }}>My best friends are:</p>
+            <ul style={{ color: `#${hex}`, fontSize: '1.2em', display: 'flex', justifyContent: 'center' }}>
+                {comp.map(color => (
+                    <li key={color.name} style={{ color: `#${color.hex}`, marginRight: '1em' }}>
```

Comment:
A lot of inline styling here, which is fine for small and simple components but gets unwieldy with more complicated stuff. Also seeing some repetitive inline-styles. You could DRY up your code and make it easier to read if you use styled components or CSS/tailwind classes.